Sem. 09/Pract.10/Task12.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+void convertString(char* str){
+    if(!str) return ;
+    while(*str){
+        if(*str >= 'a' && *str <= 'z') *str=*str-32;
+        else if(*str >='A' && *str <= 'Z') *str= *str+32;
```

Comment:
Avoid using magic numbers (32). It's better to use a constant. Also, you could avoid using a number at all (if you forget it on an exam) - `const int UPPER_TO_LOWER_DIFF = 'a' - 'A';`